src/DotNetCore.CAP.PostgreSql/CAP.PostgreSqlOptions.cs

```diff
@@ -16,6 +17,19 @@ public class PostgreSqlOptions : EFOptions
     /// Gets or sets the database's connection string that will be used to store database entities.
     /// </summary>
     public string ConnectionString { get; set; } = default!;
+    
+    /// <summary>
+    /// Gets or sets the Npgsql data source that will be used to store database entities.
+    /// </summary>
+    public NpgsqlDataSource? DataSource { get; set; }
+
+    /// <summary>
+    /// Creates an Npgsql connection from the configured data source.
+    /// </summary>
+    public NpgsqlConnection CreateConnection()
+    {
+        return DataSource != null ? DataSource.CreateConnection() : new NpgsqlConnection(ConnectionString);
+    }
```

Comment:
Is it okay to have this helper method in `PostgreSqlOptions` or should it be moved somewhere else?

Reply:
Here is ok, Should we change the accessibility from `public` to `internal`?

Reply:
Good suggestion. Done.